tony-history-server/xmlToPlayArgs.py

```diff
@@ -0,0 +1,47 @@
+import xml.etree.ElementTree as ET
+import sys
+
+# This script parses tony-site.xml, then returns a string that
+# includes all the necessary flags for Play to consume.
+# Returns empty string if there's an unrecognizable key in
+# tony-site.xml
+
+if (len(sys.argv) < 2):
+    print("Usage: python xmlToPlayArgs.py <path/to/tony-site.xml>")
```

Comment:
prefer to use lower case + underscore for python file names

Reply:
👍